### PR TITLE
Added Prometheus monitoring CPU and memory selector.

### PR DESCRIPTION
**Pull request by jinapurapu**
Added CPU and memory resource request fields for Prometheus monitoring

Screenshot from 2022-01-12 16-03-03
Screenshot from 2022-01-12 16-05-20

### CODE REVIEW

**Review by dvaldivia — CHANGES_REQUESTED** [review]
memory needs a unit, if we request `2` the value should be `2Gi` in the CRD, also if the storage class is empty don't attach it

```
  prometheus:
    diskCapacityGB: 5
    resources:
      requests:
        cpu: "2"
        memory: "2"
    storageClassName: ""
```

**Review by dvaldivia — APPROVED** [review]
LGTM. Tested.

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx [hunk 1]
```diff
@@ -129,6 +137,20 @@ const EditTenantMonitoringModal = ({
       pattern: /^[0-9]?(10)?$/,
       customPatternMessage: "Must be an integer between 0 and 10",
     });
+    tenantMonitoringValidation.push({
+      fieldKey: `newCPURequest`,
+      required: false,
+      value: newCPURequest as any as string,
+      pattern: /^[0-9]?(10)?$/,
```
Comment (cniackz) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:144]: Why the limit is 10?, where do I can see this limation coming from?

**Review by cniackz — APPROVED** [review]
Looks good to me, just have a question, why the limit of requested CPUs is 10? Can it be 11?

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx (hunk 1 above)
Comment (jinapurapu) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:144]: This limit was from a validation I reused, no actual reason. Please let me know if I should change it.

**Comment by dvaldivia** [comment]
@cniackz that is a nice catch, why are we limiting to upper 10 @jinapurapu ? it's not hat critical, but it's good to know

**Pull request merged**